docs/src/quickstart.md

```diff
@@ -232,7 +231,7 @@ prometheus-community-kube-operator        ClusterIP   443/TCP
 prometheus-community-kube-prometheus      ClusterIP   9090/TCP
 ```
 
-### Viewing with Prometheus
+### Monitoring with Prometheus
```

Comment:
I don't see an increase in clarity with this change.
Generally "monitoring" is used to mean an ongoing periodic activity.
Not 100% against, but I also don't see strong reason for it.

Reply:
I understand @jsilvela.
I would love to contribute to cnpg, are there any tips you would give someone who is a beginner to open-source contributions and cnpg to be more precise?

Reply:
Trying to improve the documentation, as you are trying to do, is a good way @Thab310 , and just to be clear, I haven't REJECTED your initiative but asked you to make changes.
If you're looking for other things to do, you might want to try with the "good first issue" items.
https://github.com/cloudnative-pg/cloudnative-pg/labels/good%20first%20issue